machine_learning/data_downloader.py

```diff
@@ -0,0 +1,167 @@
+import argparse
+import requests
+from urllib.parse import urlparse
+from web_monitoring.db import get_version_uri
+import pandas as pd
+import os
+import time
+from docopt import docopt
+
+def parse_url(url):
+    """
+    Parse Versionista change urls to get version ids
+    https://versionista.com/{site id}/{page id}/{to version id}:{from version id}
+
+    Parameters
+    ----------
+    url : string
+
+    Returns
+    -------
+    from_version_id, to_version_id : string
+        To and from version ids extracted from the url
+    """
+    if (url == ''):
+        return None
```

Comment:
Again, I was not too sure about this. While raising an error is the right thing to do, anyone using this will have to ensure that the csv data is cleaned before using it.